vignettes/programming_strategy.Rmd

```diff
@@ -739,6 +739,17 @@ Please take the following list as recommendation and try to adhere to its rules
 * `dplyr::if_else()` should be used when there are only two conditions. 
 Try to always set the `missing` argument whenever appropriate.
 
+## How Quoting is used
```

Comment:
This is a great explanatory paragraph, @zdz2101 - I am wondering if we can link to it somewhere in admiral? It would be useful for users starting to learn our framework.

Reply:
Perhaps adding a small snippet in this section right [here](https://pharmaverse.github.io/admiral/cran-release/articles/admiral.html#arguments) ?